src/components/offers/applyForm/ApplyForm.js

```diff
@@ -12,6 +12,12 @@ export const ApplyForm = () => {
     const [email, setEmail] = useState("")
     const [file, setFile] = useState(null)
 
+    const [shouldValidateField, setShouldValidateField] = useState({
+        name: false,
+        surname: false,
```

Comment:
przyjmujemy jakąś konwencje, bo chyba na backendzie i InterviewService mamy first_name i last_name

Reply:
Tu raczej nie ma to znaczenia, ale mogę potem poprawić